src/components/messages/Message/Message.scss

```diff
@@ -1,5 +1,51 @@
+$bounce-curve: cubic-bezier(0.34, 1.56, 0.64, 1);
+
 .Message {
+  $parent: &;
```

Comment:
Can you give more context as to what's going on here?

Reply:
Since using `&` would refer to its local parent instead of the root element, I am storing it in a variable called `$parent` to use it inside bubble classes to define the border radius